src/app/_components/Form/FormError.tsx

```diff
@@ -4,9 +4,14 @@ import { Description } from '@headlessui/react'
 
 export type FormErrorProps = {
   error?: string
+  hideError?: boolean
 }
 
-export function FormError({ error }: FormErrorProps) {
+export function FormError({ error, hideError }: FormErrorProps) {
+  if (hideError) {
+    return null
+  }
+
```

Comment:
I still find this an anti-pattern. The `FormError` component should be fully responsible for rendering the error message, while the parent (like `FormField`) could decide whether to pass an error or not, keeping responsibilities separate.

Can you please remind me again why we need this?

Reply:
Resolving - As we already discussed this! 🙏🏼